go/vt/vtgate/buffer/buffer.go

```diff
@@ -43,10 +44,11 @@ import (
 )
 
 var (
-	ShardMissingError    = vterrors.New(vtrpcpb.Code_UNAVAILABLE, "destination shard is missing after a resharding operation")
-	bufferFullError      = vterrors.New(vtrpcpb.Code_UNAVAILABLE, "primary buffer is full")
-	entryEvictedError    = vterrors.New(vtrpcpb.Code_UNAVAILABLE, "buffer full: request evicted for newer request")
-	contextCanceledError = vterrors.New(vtrpcpb.Code_UNAVAILABLE, "context was canceled before failover finished")
+	ShardMissingError         = vterrors.New(vtrpcpb.Code_UNAVAILABLE, "destination shard is missing after a resharding operation")
+	bufferFullError           = vterrors.New(vtrpcpb.Code_UNAVAILABLE, "primary buffer is full")
+	entryEvictedError         = vterrors.New(vtrpcpb.Code_UNAVAILABLE, "buffer full: request evicted for newer request")
+	contextCanceledError      = vterrors.New(vtrpcpb.Code_UNAVAILABLE, "context was canceled before failover finished")
+	MoveTablesInProgressError = vterrors.New(vtrpcpb.Code_UNAVAILABLE, "move tables is in the process of switching traffic")
```

Comment:
not able to find the usage of this error.

Reply:
Other errors are used to indicate that buffer cannot continue, this error looks unrelated here.

Reply:
Removed, thx.